clients/admin-ui/src/features/privacy-requests/manual-processing/ManualAccessProcessingDetail.tsx

```diff
@@ -24,18 +24,14 @@ import { PatchUploadManualWebhookDataRequest } from "privacy-requests/types";
 import React, { useRef } from "react";
 import * as Yup from "yup";
 
-import { ManualInputData } from "./types";
+import { ManualProcessingDetailProps } from "./types";
 
-type ManualAccessProcessingDetailProps = {
-  connectorName: string;
-  data: ManualInputData;
-  isSubmitting: boolean;
-  onSaveClick: (params: PatchUploadManualWebhookDataRequest) => void;
-};
```

Comment:
Exporting this from `./types` now to be able to share it with `ManualProcessingList.tsx`